workspace/src/control/control/control.py

```diff
@@ -160,8 +150,7 @@ def pub_callback(self):
         msg.throttle = np.clip(self.throttle, 0, 1)
         msg.braking = np.clip(self.braking, 0, 1)
 
-        if not self.use_sim_msg:
```

Comment:
Why was this here in the first place?

Reply:
Maybe for use art_msg type, which is for real vehicle?

Reply:
https://github.com/uwsbel/autonomy-research-testbed/blob/94e8bbbea3760f88aaed505e447e1df8db94aaf7/workspace/src/common/launch/art_launch/launch/art.launch.py#L48C33-L48C33 like the art_launch which is used to run nx, real vehicle.

Reply:
I meant more like both `art_msgs.msg.VehicleInput` and `chrono_ros_msgs.msg.ChDriverInputs` have a `header` field, so idk why we would only need to set the header stamp in sim